.github/workflows/deploy-on-label.yml

```
@@ -60,7 +60,7 @@ jobs:
         echo "Current PR state: $STATE"
     - name: Circle CI Deployment Trigger
       id: curl-circle-ci
-      if: steps.get-pr-state.outputs.STATE == 'success'
+      # if: steps.get-pr-state.outputs.STATE == 'success'
```

```suggestion
      if: steps.get-pr-state.outputs.STATE == 'success'
```

Put this back when we're done